Update Snowman Submission Page (#111)
Ada-Developers-Academy/ada-precourse-v2: intro-to-python/snowman.checkpoint.md

![new name](images/new-name.png)

You will need to complete the `snowman(snowman_word)` function using the other functions that we built in previous exercises.  

When you run the program it will give you the opportunity to run it against some automated tests by entering "t". The tests check to see if, after several guesses yor function prints out, `Congratulations, you win!` for success and `Sorry, you lose! The word was {snowman_word}` for failure.

Alternatively you can select to play the game by entering "p" and see how it works.

## Please Note

The `repl` you've cloned has several files:

1. `main.py` that will run the game. You should not modify this file.

> 1. `game.py` that holds the important `snowman(snowman_words)` function and you *will* modify

Reviewer: ```suggestion
1. `game.py` that holds the important `snowman(snowman_words)` function that you *will* modify
```

2. `game_test.py` that holds a few written test that will allow us to test your code. You should not modify this file, but feel free to take a look at what these tests expect.

Several functions have already been made for you to *use* as shown below. You do not need to create them again.

When adding your code, **you only need to modify the `snowman(snowman_word)` function**.  Note that this function takes a parameter `snowman_word`.  For testing purposes, we will be passing in the random word rather than using RandomWord.

---

> You should make use of the following functions from the previous lessons that are already created for you:

Reviewer: ```suggestion
You should make use of the following functions from the previous lessons which have been created for you:
```
